tools/tx_pull.py

```diff
@@ -107,13 +108,62 @@ def get_parser():
     parser.add_argument(
         '-e', '--email', dest='email',
         help=('Provides an email address used to commit on github if the one '
-              'associated to the github account is not pulbic'))
+              'associated to the GitHub account is not public'))
     parser.add_argument(
         '-t', '--target-org', dest='target',
-        help=('By default, translation are committed in github on OCA. This '
+        help=('By default, translation are committed in GitHub on OCA. This '
               'arg lets you provide an alternative org'))
     return parser
 
+
+def wrap_tx_call(func, args=None, kwargs=None):
```

Comment:
What is wrong with the first argument?

<img width="464" alt="screen shot 2016-11-29 at 2 52 34 pm" src="https://cloud.githubusercontent.com/assets/6644187/20728393/24f8a542-b643-11e6-8bd8-f9aef30695d0.png">